libs/sdk-core/src/persist/migrations.rs

```diff
@@ -428,6 +428,18 @@ pub(crate) fn current_migrations() -> Vec<&'static str> {
         "
         ALTER TABLE swaps_info ADD COLUMN channel_opening_fees TEXT;
        ",
+        // Swaps synchronization: Add sync table that stores the fees used in swaps
+        "
+        CREATE TABLE IF NOT EXISTS sync.swaps_fees (
+         bitcoin_address TEXT PRIMARY KEY NOT NULL,
+         created_at TEXT DEFAULT CURRENT_TIMESTAMP NOT NULL,
+         channel_opening_fees TEXT NOT NULL
+        ) STRICT;
+        ",
+        // Remove dynamic fee from swaps_info (fees already stored in swaps_fees)
+        "
+        ALTER TABLE swaps_info DROP COLUMN channel_opening_fees;
```

Comment:
@ok300  If we didn't merge this branch and the feature yet then we can remove the migration line that added the channel_opening_fees to the swaps_info table and remove this line that drops it.

Reply:
Good catch, done